common/src/commonMain/kotlin/io/xorum/codeforceswatcher/features/contests/models/Contest.kt

```diff
@@ -0,0 +1,23 @@
+package io.xorum.codeforceswatcher.features.contests.models
+
+import io.xorum.codeforceswatcher.Contest
+
+data class Contest(
+        val id: Long,
+        val name: String,
+        val startTimeSeconds: Long,
+        val durationSeconds: Long,
+        val phase: String
+) {
+    companion object {
+        fun fromDB(dbContest: Contest): io.xorum.codeforceswatcher.features.contests.models.Contest {
```

Comment:
Can be written like `fun fromDB(dbContest: Contest) = io.xorum.codeforceswatcher.features.contests.models.Contest(`